opennlp-tools/src/test/java/opennlp/tools/namefind/NameFinderMETest.java

```diff
@@ -109,6 +110,21 @@ public void testNameFinder() throws Exception {
     Assert.assertEquals(new Span(4, 6, DEFAULT), names[1]);
   }
 
+  @Test
+  public void testNameFinderWithDownloadedModel() throws Exception {
+
+    String input = "Pierre Vinken , 61 years old , will join the board as a nonexecutive director Nov. 29 .";
+    String[] sentence = input.split(" ");
+
+    TokenNameFinder nameFinder = new NameFinderME("en", DownloadUtil.EntityType.PERSON);
```

Comment:
I wonder if some of these tests could become flaky if there are internet connection issues in the client or the server side?

Reply:
That's a good point. I'll see if I can make those integration tests that aren't run with every build.

Reply:
Good idea, or group them so that if one starts failing we can disable temporarily until whatever issue is fixed and they can be re-enabled.

Reply:
Updated and move those tests to *IT.java tests outside the regular build.